src/util/hass-translation.ts

```diff
@@ -41,9 +40,8 @@ function findAvailableLanguage(language: string) {
 /**
  * Get user selected language from backend
  */
-export async function getUserLanguage(hass: HomeAssistant) {
-  const result = await fetchTranslationPreferences(hass);
-  const language = result ? result.language : null;
```

Comment:
Why was this logic moved?

Reply:
@bramkragten this was done to avoid fetching data twice as mentioned in https://github.com/home-assistant/frontend/pull/7925#pullrequestreview-579235645. At this point we can skip using `getUserLanguage()` and just use `findAvailableLanguage()` instead. The actual fetch from the backend needs to return the entire object with the `language` and `number_format` and is done in the `translations-mixin.ts` file now. `getUserLanguage()` was only used once in the codebase and does not seem to be needed as a distinct function. I'll push a new commit to make this change.

Reply:
But why not just add the `number_format` to the existing function `getUserLanguage` and rename that function to `getUserLocale`?

Reply:
Done!